ton_client/client/src/contracts/deploy.rs

```diff
@@ -24,96 +25,116 @@ use ton_sdk::NodeClient;
 
 const DEFAULT_WORKCHAIN: i32 = 0;
 
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct DeployFunctionCallSet {
+    pub abi: serde_json::Value,
+    pub constructor_header: Option<serde_json::Value>,
```

Comment:
Can you rename it to shorter form: `header` and `params`?

Reply:
No. Bindings use these names